setup.py

```diff
@@ -29,6 +29,8 @@
         'License :: OSI Approved :: BSD License',
         'Topic :: Software Development :: Libraries :: Python Modules'],
 
+    install_requires=("django_apptemplates",),
```

Comment:
One of the benefits of overwriting django-apptemplates would be to avoid the drawback of not being able to just choose one that does the job best.